src/main/kotlin/org/kamiblue/client/module/modules/combat/AutoMend.kt

```diff
@@ -62,6 +68,8 @@ internal object AutoMend : Module(
 
         onDisable {
             switchback()
+            if (AutoArmor.isPaused && pauseAutoArmor)
+                AutoArmor.isPaused = false
```

Comment:
I'm thinking about possible conflicts here, what if someone disables pauseAutoArmor while it's paused? Add a setting listener to pauseAutoArmor to disable AutoArmor.isPaused when the setting is disabled